src/styles/_List.scss

```diff
@@ -57,11 +57,11 @@
 .list-letter {
   display: inline-block;
   height: 40px;
-  width: 40px;
+  min-width: 40px;
   margin-right: 20px;
   line-height: 40px;
   text-align: center;
-  border-radius: 40px;
+  border-radius: 50%;
```

Comment:
are the letter circles actually smaller (or larger for that matter) on mobile? How does that jive with the icons when we have them?

Reply:
I can leave the 40px fixed size. It's just a % works for any size element.
Should leave the size in pixels?

Reply:
I'm mostly interested in whether or not the size is changing. For example, the change from `width` to `min-width` and making the radius proportional might indicate that in some cases the width would be larger than 40px. Is that the intention? 

Related, should the change to use `min-width` be accompanied by a change in the use of `height`?

In the end I'm not really sure why these changes are needed or the intent

Reply:
> I'm mostly interested in whether or not the size is changing. For example, the change from `width` to `min-width` and making the radius proportional might indicate that in some cases the width would be larger than 40px. Is that the intention?

If we set it to `width: 40px ` the icon will be smaller on mobile size screens. Like this:

<img width="418" src="https://user-images.githubusercontent.com/899175/53905560-8afee500-4051-11e9-88a6-db783b2081b3.png">

> Related, should the change to use `min-width` be accompanied by a change in the use of `height`?

Looks like the height of this `<span>` is determined by the `line-height: 40px;`
But the `height` style limits its height so it doesn't grow over the 40px:

![mar-06-2019 20-56-02](https://user-images.githubusercontent.com/899175/53905903-53dd0380-4052-11e9-9225-a60b10388bc9.gif)


> In the end I'm not really sure why these changes are needed or the intent

It was merely to fix the Maven icon on Mobile size screens so it doesn't look like it has shirked